recipes-devtools/python/python-catkin-pkg.inc

```diff
@@ -9,4 +9,7 @@ SRC_URI[sha256sum] = "d4d069eaeb29d6a9ee71773b20c41515b6e0feb9ea2765ebaf3e599d0c
 
 inherit pypi
 
-BBCLASSEXTEND = "native"
+RDEPENDS_${PN} = "${PYTHON_PN}-unixadmin"
+RDEPENDS_${PN}_class-native = ""
+
+BBCLASSEXTEND = "native nativesdk"
```

Comment:
Don't we need another line under it:FILES_${PN}_append_class-nativesdk = " ${SDKPATHNATIVE}"